.chronus/changes/improvements-for-the-intellisense-of-tspconfig.yaml-2024-11-19-2-35-1.md

```diff
@@ -0,0 +1,10 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: fix
+packages:
+  - "@typespec/compiler"
+---
+
+Improvements for the intellisense of tspconfig.yaml
+- Support the auto completion for extends, imports, rule, rule sets and variables in tspconfig.yaml
```

Comment:
split those 2 just as 2 different change file, this will look off otherwise in the generated changelog